spec/models/agents/google_translation_agent_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
+
+describe Agents::GoogleTranslationAgent do
+    before do
```

Comment:
We generally use two space indentation in ruby files, looks like your editor choose to use 4 here 😄